tests/test_named_spatial_relationships.py

```diff
@@ -0,0 +1,238 @@
+import math
+
+import geopandas as gpd
+import pytest
+import shapely
+
+import geofileops as gfo
+
+zones = [
+    "POLYGON ((0 0, 3 0, 3 3, 0 3, 0 0))",
+    "POLYGON ((10 1, 13 1, 13 4, 10 4, 10 1))",
+]
+
+extra_parcels = [
+    "POLYGON ((-1 5, 1 5, 1 7, -1 7, -1 5))",
+    "POLYGON ((3 6, 5 6, 5 8, 3 8, 3 6))",
+]
+
+
+@pytest.mark.parametrize("zones", [zones])
+@pytest.mark.parametrize(
+    "parcels", [["POLYGON ((2 1, 4 1, 4 3, 2 3, 2 1))", *extra_parcels]]
+)
+@pytest.mark.parametrize("subdivide_coords", [0, 10])
+@pytest.mark.parametrize(
+    "spatial_relations, exp_features",
+    [("intersects is True", 1), ("intersects is False", 2)],
+)
+def test_intersects(
```

Comment:
Maybe it is clearer to name this module `test_export_by_location.py` and to put all tests related to `export_by_location` 
 (also the one(s) now in test_geofileops_twolayers.py) together here?